Tasks/AzureResourceGroupDeployment/task.json

```diff
@@ -14,7 +14,7 @@
     "version": {
         "Major": 2,
         "Minor": 1,
-        "Patch": 8
+        "Patch": 9
```

Comment:
can you build the task once. since the task.loc.json should also have the new version